### ISSUE TITLE
Install library manually

### ISSUE THREAD

**Issue by kostas-baltsoft** [ConvertAPI/convertapi-php#47]
Our readme file lacks a complete description of how to install the library without using Composer:

https://www.convertapi.com/doc/php-library#manual-installation

Users are asking about this line:

```php
require_once('/path/to/convertapi-php/src/ConvertApi/autoload.php');
```

What is the ```/path/to/convertapi-php/```? Do they need to copy the contents of this repo into their project?

**Comment by Jan-E** [comment]
Looks like the autoload.php is convertapi-php/**lib**/ConvertApi/autoload.php if you clone this repo,
not convertapi-php/**src**/ConvertApi/autoload.php
https://github.com/ConvertAPI/convertapi-php/blob/master/lib/ConvertApi/autoload.php

**Comment by laurynas-baltsoft** [comment]
Fixed in 2.0.0 version:
https://github.com/ConvertAPI/convertapi-php/releases/tag/v2.0.0
